src/home-page/NoMatch404.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import Typography from '@material-ui/core/Typography';
+import styled from 'styled-components';
+import { Button } from '@material-ui/core';
+import { MainContainer } from '../layout/components/MainContainer';
+
+const NoMatchTypography = styled('div')`
+  display:flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction:column;
+  margin: auto;
+  min-height: 70vh;
+`;
+
+const HeaderTypography = styled(Typography)`
+  font-size:10em;
```

Comment:
I know it seems like a very small thing, but could we add a space between css property and value (also for SubtitleTypography)?

Reply:
done